src/main/scala/com/redis/IO.scala

```diff
@@ -73,7 +77,7 @@ trait IO extends Log {
         os.write(data)
         os.flush
       } catch {
-        case x: Throwable => throw new RedisConnectionException("connection is closed. write error")
+        case x: Throwable => throw new RedisConnectionException(s"connection error. write error: ${x.getMessage}")
```

Comment:
we do. This is the write timeout. the connection one is higher up than this. Do you mean a different exception class?